packages/common_client/lib/src/ld_common_client.dart

```diff
@@ -221,19 +221,19 @@ final class LDCommonClient {
     // TODO: Do we start the process when we create the client, and provide
     // a way to know when it completes? Or do we not even start it as we
     // are doing here.
-    _envReporter = await _makeEnvReporter();
+    _envReport = await _makeEnvironmentReport();
```

Comment:
Is this running in all cases now?  Before I believe if a customer had auto env attributes turned off, the env code would only query for the osInfo to use in the diagnostic portion down below.

Reply:
It does. We should make sure that we are clear that this information only is contingent on going into the context, not that we don't call the APIs.